pkgs/development/python-modules/pdfrw/default.nix

```diff
@@ -0,0 +1,58 @@
+{ stdenv, lib, buildPythonPackage, fetchFromGitHub, fetchPypi, isPyPy, python3, pytest, zlib }:
+
+with python3.pkgs;
```

Comment:
Its better to list all dependencies in the function that returns the derivation.